packages/types/src/deletedObject.ts

```diff
@@ -1,6 +1,4 @@
-import type { ClerkResource } from './resource';
-
-export interface DeletedObjectResource extends ClerkResource {
+export interface DeletedObjectResource {
```

Comment:
@dimkl This was added by me in a commit some days ago, and TypeScript is not happy about it because of how this implements some of the properties. I thought `DeletedObjectResource` might be a special resource, so I reverted to what it was before.

Reply:
ok. i think we need note and investigate about what happens in resources since we have to enforce that all of them somehow extend a base resource interface (comment unrelated to this PR).